examples/python/Quantinuum_emulators.py

```diff
@@ -48,7 +48,7 @@
 
 print(machine, "status:", backend.device_state(device_name=machine))
 
-# Compile the circuit to the Quantinuum backend with `get_compiled_circuit`. See the `pytket` [User Manual](https://tket.quantinuum.com/user-manual/index.html) for more information on all the options that are available.
```

Comment:
Did you mean to change this link?

Reply:
I think it just changed where it is in the notebook. The link is still there, I can see it to the `get_compiled_circuit` function